### DISCUSSION TITLE
My opinion 😊.

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#250] [General]
Hi, I'm 16 years old and I love this proyect. I hope in the development of this. I would like them to introduce the option of rounded bottom corners and a dark theme. Thank you for your hard work. I feel like I will love helloSystem.

My hug for helloSystem.❤ ¡I'm so exciting!

**Comment by probonopd** [comment]
Thank you very much @alphamodh0 - glad you like it! Now, have a look at how it works under the hood... and start making the things you'd like it to have.

**Reply by alphamodh0** [reply]
I don't know much about programming but I will put all my efforts to improve helloSystem. I am an immigrant from Linux and I have never seen such a perfect hedge over what I have always sought. I hope that when I graduate from computer engineering I can be someone like you.

**Reply by probonopd** [reply]
All the power to you! You can help the project not just by programming, but also by testing. We are always interested in what works on what machines and what doesn't. Have fun!
